dts/bindings/adc/lltc,ltc2451.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Brill Power Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Linear Technology LTC2451 ADC
+
+compatible: "lltc,ltc2451"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+  conversion-speed:
+    type: int
+    default: 0
```

Comment:
The reason for choosing this default value needs to be justified in the description

Reply:
The default was only added as that's what was most convenient for my application initially, has been removed.